apps/valkyrie/test/integration/valkyrie_test.exs

```diff
@@ -1,16 +1,19 @@
 defmodule ValkyrieTest do
   use ExUnit.Case
   use Divo
+  use Properties, otp_app: :valkyrie
+
   alias SmartCity.TestDataGenerator, as: TDG
   import SmartCity.TestHelper
   import SmartCity.Event, only: [data_ingest_start: 0]
   alias TelemetryEvent.Helper.TelemetryEventHelper
 
-  @endpoints Application.get_env(:valkyrie, :elsa_brokers)
-  @dlq_topic Application.get_env(:dead_letter, :driver) |> get_in([:init_args, :topic])
-  @input_topic_prefix Application.get_env(:valkyrie, :input_topic_prefix)
-  @output_topic_prefix Application.get_env(:valkyrie, :output_topic_prefix)
   @instance_name Valkyrie.instance_name()
+  @dlq_topic Application.get_env(:dead_letter, :driver) |> get_in([:init_args, :topic])
```

Comment:
Is this sort of deep get something that Properties could support? (Maybe it already does?)